pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
```

Comment:
Ok so you're talking about concrete use-cases as opposed to a generalized pattern, right? I think that makes sense, but isn't enough to warrant the "example:" prefix. I believe the phrase itself is (or should be) descriptive enough to make the distinction clear to users.